cos_alerter/alerter.py

```diff
@@ -30,15 +33,29 @@ def set_path(self, path: str):
         """Set the config file path."""
         self.path = Path(path)
 
+    def _hash_keys(self, clients):
+        """Hash the keys in the clients dictionary."""
+        for client_info in clients.values():
+            client_key = client_info.get("key", "")
+            if client_key:
+                hashed_key = hashlib.sha256(client_key.encode()).hexdigest()
+                client_info["key"] = hashed_key
+
     def reload(self):
         """Reload config values from the disk."""
+        yaml = YAML(typ="rt")
         with open(
             os.path.join(os.path.dirname(os.path.realpath(__file__)), "config-defaults.yaml")
         ) as f:
-            self.data = yaml.safe_load(f)
+            self.data = yaml.load(f)
         with open(self.path, "r") as f:
-            user_data = yaml.safe_load(f)
+            try:
+                user_data = yaml.load(f)
+            except DuplicateKeyError:
+                logger.error("Duplicate client IDs found in COS Alerter config. Exiting...")
```

Comment:
This should be `critical` rather than `error`. See [here](https://docs.python.org/3/library/logging.html#logging-levels)